assets/i18n/en.json

```diff
@@ -93,11 +93,11 @@
     "dgc-community-modal-not-bite": "Try us, we promise we won't bite!", 
     "dgc-exchanges-modal-title": "GET DOGECOIN:",
     "dgc-exchanges-modal-many-ways": "There are numerous amounts of ways to get Dogecoin. You can buy them, trade for them, get tipped, \"mine\" them, and more. They are surprisingly endless!", 
-    "dgc-exchanges-modal-tipped": "You can get tipped Dogecoin by participating in <a class=\"btn btn-sm btn-outline-secondary\" target=\"_blank\" href=\"https://www.reddit.com/r/dogecoin\">our community</a>.<br> If that's not your style, the easiest way to have your first Dogecoin is to get them from a faucet.", 
-    "dgc-exchanges-modal-faucet-faq": "A Faucet is a website which gives you a small amount of Dogecoin for free to introduce you to the currency! All you need is your Dogecoin address.", 
-    "dgc-exchanges-modal-faucet-sites": "Here are a couple Dogecoin faucets:", 
-    "dgc-exchanges-modal-dogefaucet": "Dogefaucet.com", 
-    "dgc-exchanges-modal-dogefaucet-desc": " - Enter your address to get free Dogecoin.", 
+    "dgc-exchanges-modal-tipped": "You can get tipped Dogecoin by participating in <a class=\"btn btn-sm btn-outline-secondary\" target=\"_blank\" href=\"http://www.reddit.com/r/dogecoin\">our community</a>. If that's not your style, another way to have your first Dogecoin is to participate at Folding@Home.", 
+    "dgc-exchanges-modal-fah-faq": "<a class=\"btn btn-sm btn-outline-secondary\" target=\"_blank\" href=\"https://www.dogecoinfah.com\" target=\"_blank\">Folding@Home (FaH)</a> is a distributed computing project. You can join team dogefolders, bring in your computing power, help to find find cures for diseases and receive some Doges for your efforts.", 
+    "dgc-exchanges-modal-faucet-sites": "Here are a all resources to start folding:", 
+    "dgc-exchanges-modal-dogecoinfah": "DogecoinFaH.com", 
+    "dgc-exchanges-modal-dogecoinfah-desc": " - Team dogefolders.", 
```

Comment:
I think those tags are no longer implemented in the html. Can we remove them? Also for other translations.